client-ts/Microsoft.AspNetCore.SignalR.Client.TS/HubConnection.ts

```diff
@@ -310,4 +317,11 @@ export class HubConnection {
             arguments: args,
         };
     }
+
+    private createCancelInvocation(id: string): CancelInvocation {
+        return <CancelInvocation>{
```

Comment:
Do you need the cast? I thought TypeScript would be smart enough to infer this. If it's necessary though, keep it.